package-res/resources/web/pentaho/type/Context.js

```diff
@@ -155,15 +240,63 @@ define([
     },
 
     /**
-     * Gets the instance constructor of a type.
+     * Gets, asynchronously, the **configured instance constructor** instance constructor of a type.
+     *
+     * For more information on the `typeRef` argument,
+     * please see [UTypeReference]{@link pentaho.type.spec.UTypeReference}.
+     *
+     * This method can be used even if a generic type metadata specification references
+     * non-standard types whose modules have not yet been loaded by the AMD module system.
+     *
+     * An error is thrown when:
```

Comment:
No I don't. I simply didn't use in the remainder of the code, and didn't feel like starting to use it now. I guess I am subconsciously relying on a subsequent pass through all the code to fix that.
An interesting case arises in this use, though: that rejections and throws, being similar conditions, would be documented differently, if we were to use the `@throws` clause.